src/main/java/com/github/steveice10/mc/protocol/data/message/MessageSerializer.java

```diff
@@ -121,6 +123,21 @@ public static JsonElement toJson(Message message) {
             } else {
                 throw new IllegalArgumentException("Unknown NBT message type in json: " + json);
             }
+        } else if(json.has("type") && json.has("id")) {
+            EntityHoverMessage.Builder builder = new EntityHoverMessage.Builder();
+            builder.type(json.get("type").getAsString());
+            builder.id(json.get("id").getAsString());
+            if (json.has("name"))
```

Comment:
Please explicitly use brackets. Ditto for the two occurrences below.

Reply:
Fixed